toolchain/gcc/PKGBUILD

```diff
@@ -121,11 +121,17 @@ build() {
   export CXXLAGS+=" -fno-semantic-interposition"
 
   if [ -n "$_optimize_v3" ]; then
-    export ASFLAGS="-D__AVX__=1 -D__AVX2__=1 -msse2avx -D__FMA__=1 "
-  elif [-n "$_optimize_v4" ]; then
-    export ASFLAGS="-D__AVX__=1 -D__AVX2__=1 -D__AVX512__=1 -msse2avx -D__FMA__=1 "
+    export ASFLAGS+="-D__AVX__=1 -D__AVX2__=1 -msse2avx -D__FMA__=1 "
```

Comment:
Since we are now using `+=`, we need to add a leading space at the beginning so that we don't get into trouble if `ASFLAGS` are already specified somewhere in `makepkg.conf` and don't have a space after them.

Reply:
right, thanks